documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -31,10 +31,10 @@ There could be various reasons to prefer this over an "embedded Lucene" approach
 * benefit from Elasticsearch's out of the box horizontal scalability features
 * explore the data updated by an Hibernate powered application using the Elasticsearch dashboard integrations such as Kibana
 
-There are a couple drawbacks compared to the embedded Lucene approach though:
+There are a couple of drawbacks compared to the embedded Lucene approach though:
 
 * incur a performance penalty of remote RPCs both for index updates and to run queries
-* need to buy and manage additional servers
+* need to manage an additional service
```

Comment:
Yes you can do it, but it will require a higher performance server just to deal with the several dozens of additional threads. And you'll need some more for the efficiency lost in the transport & serializations of requests and responses..